micrometer/pom.xml

```diff
@@ -50,6 +50,17 @@
       <version>2.0.0.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.cloud</groupId>
+      <artifactId>spring-cloud-openfeign-core</artifactId>
+      <version>2.2.6.RELEASE</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Please use our internal contract for testing.  We cannot rely on 3rd party contracts.

Reply:
modified!